jaxley/integrate.py

```diff
@@ -366,6 +371,15 @@ def integrate(
     )
     all_states, all_params = init_fn(params, all_states, param_state, delta_t)
 
+    if solver == "exp_euler":
```

Comment:
if this is still experimental, maybe we should emit a warning and ask users to report bugs.